### PR TITLE
Commented "empty" solution file for students

### PR DESCRIPTION
**Pull request by kotp**
Reference: #987, #982 and #541

### CODE REVIEW

File: exercises/isbn-verifier/isbn_verifier.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+=begin
+Write your code for the 'Isbn Verifier' exercise in this file. Make the tests in
```
Comment (pgaspar) [inline exercises/isbn-verifier/isbn_verifier.rb]: Consider:

```suggestion
Write your code for the 'ISBN Verifier' exercise in this file. Make the tests in
```

File: exercises/etl/etl.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+=begin
+Write your code for the 'Etl' exercise in this file. Make the tests in
```
Comment (pgaspar) [inline exercises/etl/etl.rb]: Consider:

```suggestion
Write your code for the 'ETL' exercise in this file. Make the tests in
```

**Comment by kotp** [comment]
Thank you @pgaspar for the nice touch up.

**Comment by pgaspar** [comment]
@kotp feel free to merge, unless you want to wait for more feedback.

**Pull request merged**